roles/upgrade/post-upgrade/defaults/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 # how long to wait for cilium after upgrade before uncordoning
 upgrade_post_cilium_wait_timeout: 120s
+upgrade_node_post_upgrade_confirm: false
```

Comment:
Hi @mac-chaffee , thanks for adding this functionality!
Please add also some notes in our documentation for this so it is obvious to users and easy to discover.